src/App.css

```diff
@@ -57,3 +73,111 @@
 .last-row {
   grid-template-columns: 2fr 1fr 1fr;
 }
+
+/* Stylize and positioning NavBar */
+nav {
+  padding: 0.8rem;
+  display: flex;
```

Comment:
### OPTIONAL
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and css best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)